bots/models.py

```
@@ -0,0 +1,2243 @@
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    price_multiplier = models.FloatField(default=1)
+    emoji = models.TextField(blank=True, default="")
+
+    def __str__(self):
+        return self.meta_title
+
+
+class SavedRunQuerySet(models.QuerySet):
+    def to_df(self, tz=pytz.timezone(settings.TIME_ZONE)) -> "pd.DataFrame":
```

Ruff (0.8.2)

Undefined name `pd`

(F821)